charts/victoria-metrics-k8s-stack/hack/sync_rules.py

```diff
@@ -75,7 +75,13 @@ def new_representer(dumper, data):
 condition_map = {
     'etcd': '.Values.kubeEtcd.enabled .Values.defaultRules.rules.etcd',
     'general.rules': '.Values.defaultRules.rules.general',
-    'k8s.rules': '.Values.defaultRules.rules.k8s',
+    'k8s.rules.container_cpu_usage_seconds_total': '.Values.k8sContainerCpuUsageSecondsTotal.enabled .Values.defaultRules.rules.k8sContainerCpuUsageSecondsTotal',
+    'k8s.rules.container_memory_cache': '.Values.k8sContainerMemoryCache.enabled .Values.defaultRules.rules.k8sContainerMemoryCache',
+    'k8s.rules.container_memory_rss': '.Values.k8sContainerMemoryRss.enabled .Values.defaultRules.rules.k8sContainerMemoryRss',
+    'k8s.rules.container_memory_swap': '.Values.k8sContainerMemorySwap.enabled .Values.defaultRules.rules.k8sContainerMemorySwap',
+    'k8s.rules.container_memory_workingSetBytes': '.Values.k8sContainerMemoryWorkingSetBytes.enabled .Values.defaultRules.rules.k8sContainerMemoryWorkingSetBytes',
+    'k8s.rules.container_resource': '.Values.k8sContainerResource.enabled .Values.defaultRules.rules.k8sContainerResource',
+    'k8s.rules.pod_owner': '.Values.k8sPodOwner.enabled .Values.defaultRules.rules.k8sPodOwner',
```

Comment:
What is `.Values.k8sContainerCpuUsageSecondsTotal.enabled` used for? Wouldn't `.Values.defaultRules.rules.k8sContainerCpuUsageSecondsTotal` alone be just fine? (same for all other lines

Reply:
I was looking at the kubapi rule. I removed the double function

Reply:
Ah right in these cases I believe the double conditionals are to disable alerting rules when metric collection is disabled for a specific service.

 Let's run `sync_rules.py` one more time to sync the `charts/victoria-metrics-k8s-stack/templates/rules/*.yaml` rules once more.

Reply:
Done. thank you for your patience 😊